content/docs/overview/scopes-and-threading.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Scopes and Threading"
+---
+## Introduction
+
+The OpenTracing API allows for only one `span` in a thread to be active at any point in time. There can be other spans involved with the same thread, which satisfy the following conditions:
```

Comment:
It would be better to start with an introduction to the Active Span concept, then mention scopes, then get into these details.